<launch>
    
    <group ns="/histogram_segmentation">
        <rosparam command="load" file="$(find pal_vision_segmentation)/default_tabletop_histogram.yaml" />
    </group>

    <node pkg="pal_vision_segmentation" type="histogram_segmentation" name="histogram_segmentation" args="$(find pal_vision_segmentation)/etc/manzanilla1.jpg">
      <remap from="image" to="/stereo/left/image" />
    </node>

</launch>

